src/NuGet.Services.Metadata.Catalog.Monitoring/Resources/PackageBaseAddressResource.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class PackageBaseAddressResource : INuGetResource
```

Comment:
You're probably wondering - why did I create a `INuGetResource` just so I could pass the `packageBaseAddress` config to the `FlatContainerValidator`? `Validator`s are created through reflection by the `ValidatorFactory`, and this factory only supports one kind of constructor for `Validator`s. Ideally, `Validator`s would be created through dependency injection, but this would be a large refactoring.